src/sql/workbench/contrib/notebook/browser/notebook.css

```diff
@@ -6,7 +6,11 @@
 	margin-top: 5px;
 }
 
-.notebookEditor .editor-toolbar {
```

Comment:
It looks like this style is only being applied when preview features are enabled now - is that intentional?

Reply:
My mistake. That should apply in both cases.